src/app/pages/details/details.hbs

```diff
@@ -0,0 +1,172 @@
+                anim. Andouille sunt labore sint, incididunt turkey tri-tip
+                swine pork chop jowl cow fatback spare ribs shank enim. Commodo
+                cow laborum ad adipisicing spare ribs excepteur.
+            </p>
+        </div>
+        <div class="blurb">
+            <img src="https://placeholdit.imgix.net/~text?txtsize=33&txt=Logo&w=200&h=45" />
+            <h4>Sapling Learning</h4>
+            <p class="text">
+                Bacon ipsum dolor amet incididunt aliqua salami tongue fugiat
+                anim. Andouille sunt labore sint, incididunt turkey tri-tip
+                swine pork chop jowl cow fatback spare ribs shank enim. Commodo
+                cow laborum ad adipisicing spare ribs excepteur.
+            </p>
+        </div>
+        <div class="blurb">
+            <img src="https://placeholdit.imgix.net/~text?txtsize=33&txt=Logo&w=200&h=45" />
+            <h4>Sapling Learning</h4>
+            <p class="text">
+                Bacon ipsum dolor amet incididunt aliqua salami tongue fugiat
+                anim. Andouille sunt labore sint, incididunt turkey tri-tip
+                swine pork chop jowl cow fatback spare ribs shank enim. Commodo
+                cow laborum ad adipisicing spare ribs excepteur.
+            </p>
+        </div>
+    </div>
+    <h3>Adaptive Courseware</h3>
+    <div class="resource-row">
+        <div class="blurb">
+            <img src="https://placeholdit.imgix.net/~text?txtsize=33&txt=Logo&w=200&h=45" />
+            <h4>Sapling Learning</h4>
+            <p class="text">
+                Bacon ipsum dolor amet incididunt aliqua salami tongue fugiat
+                anim. Andouille sunt labore sint, incididunt turkey tri-tip
+                swine pork chop jowl cow fatback spare ribs shank enim. Commodo
+                cow laborum ad adipisicing spare ribs excepteur.
+            </p>
+        </div>
+        <div class="blurb">
+            <img src="https://placeholdit.imgix.net/~text?txtsize=33&txt=Logo&w=200&h=45" />
+            <h4>Sapling Learning</h4>
+            <p class="text">
+                Bacon ipsum dolor amet incididunt aliqua salami tongue fugiat
+                anim. Andouille sunt labore sint, incididunt turkey tri-tip
+                swine pork chop jowl cow fatback spare ribs shank enim. Commodo
+                cow laborum ad adipisicing spare ribs excepteur.
+            </p>
+        </div>
+        <div class="blurb">
+            <img src="https://placeholdit.imgix.net/~text?txtsize=33&txt=Logo&w=200&h=45" />
+            <h4>Sapling Learning</h4>
+            <p class="text">
+                Bacon ipsum dolor amet incididunt aliqua salami tongue fugiat
+                anim. Andouille sunt labore sint, incididunt turkey tri-tip
+                swine pork chop jowl cow fatback spare ribs shank enim. Commodo
+                cow laborum ad adipisicing spare ribs excepteur.
+            </p>
+        </div>
```

Comment:
Here's another good place for a model rather than repeating these identical blobs of code numerous times.